app/routes/session.py

```diff
@@ -128,9 +151,76 @@ def login() -> Response:
 
     cookie_session["language_code"] = metadata.language_code
 
+    if (dataset_id := metadata["sds_dataset_id"]) and ru_ref:
+        get_supplementary_data(
+            supplementary_data_url=SUPPLEMENTARY_DATA_URL,
+            dataset_id=dataset_id,
+            ru_ref=ru_ref,
+        )
+
     return redirect(url_for("questionnaire.get_questionnaire"))
 
 
+def get_supplementary_data(
+    supplementary_data_url: str, dataset_id: str, ru_ref: str
```

Comment:
does the url need to be passed in as a param or could this function use the constant?

Reply:
I just thought it was clearer for now as not sure how the actual SDS endpoint will be passed/stored, assuming an env var like the other URLs

Reply:
yeah that makes sense, works as is 👍

Reply:
It will be an env var, so I suggest not passing it. Use it via `current_app.config["SDS_API_BASE_URL"]`.